docs/usage.md

```diff
@@ -3,7 +3,27 @@
 This is lazy dog HOWTO to using CodeChecker analysis.
 It invokes Clang Static Analyzer and Clang-Tidy tools to analyze your code.
 
-## Step 1: Integrate CodeChecker into your build system
+Table of Contents
+=================
+* [Step 1: Integrate CodeChecker into your build system](#step-1)
+* [Step 2: Analyze your code](#step-2)
+  * [Cross-Compilation](#cross-compilation)
+  * [Incremental Analysis](#incremental-analysis)
+  * [Analysis Failures](#analysis-failures)
+* [Step 3: Store analysis results in a CodeChecker DB and visualize results](#step-3)
+* [Step 4: Fine tune Analysis configuration](#step-4)
+  * [Ignore modules from your analysis](#ignore-modules)
+  * [Enable/Disable Checkers](#enable-disable-checkers)
+  * [Identify files that failed analysis](#identify-files)
+* [Step 5: Integrate CodeChecker into your CI loop](#step-5)
+  * [Storing daily runs](#storing-daily-runs)
+  * [Store each analysis in a new run](#storing-new-runs)
+    * [Example](#storing-new-runs-example)
```

Comment:
Do we want to include `Example` into the table of content? I've seen it at multiple places but I'm not sure we should include them.

Reply:
It's a direct link to a code snippet only... well it's debateable. Including it makes the TOS easier, and perhaps later on, automatic to maintain... but I don't see a reason in including this specifically either.

Reply:
If the user knows about the feature but don't know how to use it, he checks the TOC and sees that there is an example for this feature and he can jump to it. I think we should leave it in the TOC.